### PR TITLE
po: update Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by rffontenelle**

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -809,46 +901,60 @@ msgstr ""
...
+msgstr ""
+"É necessária autenticação para congelar ou descongelar os processos da "
+"unidade “$(unit)”."
+
+#~ msgid ""
+#~ "Authentication is required to halt the system while an application asked "
+#~ "to inhibit it."
+#~ msgstr ""
+#~ "É necessária autenticação para parar o sistema enquanto um aplicativo "
+#~ "solicitou inibição."
```
Comment (keszybz) [inline po/pt_BR.po]: Please drop any commented-out entries.

Comment (rffontenelle) [reply po/pt_BR.po]: I don't think that's a good idea... While representing zero impact to the software itself, these obsolete strings may be automatically used at some time in the future by Gettext to pre-fill newly added strings as fuzzy.

For instance, see [gnu.org's recent-releases-include PO file] for a file that has obsolete strings which gets used, and are very useful to me on after update.

Do you still want me to remove it?

[gnu.org's recent-releases-include PO file]: https://www.gnu.org/software/po/recent-releases-include.pt-br.po

Comment (keszybz) [reply po/pt_BR.po]: In this particular case, the original string was poorly worded, and it seems very unlikely that we'll re-include it. The translation for the reworded string is present in the po file, and that should be used in preference. So yeah, please remove it.

Comment (rffontenelle) [reply po/pt_BR.po]: Done.

Comment (keszybz) [reply po/pt_BR.po]: Thanks.

**Pull request merged**